deps/gtest/src/gtest.cc

```diff
@@ -3596,21 +3596,23 @@ void TapUnitTestResultPrinter::OutputTapTestInfo(int* count,
   *stream << "  ---\n";
   *stream << "  duration_ms: " <<
              FormatTimeInMillisAsSeconds(result.elapsed_time()) << "\n";
-  *stream << "  ...\n";
 
-  for (int i = 0; i < result.total_part_count(); ++i) {
-    const TestPartResult& part = result.GetTestPartResult(i);
-    OutputTapComment(stream, part.message());
+  if (result.total_part_count() > 0) {
+    for (int i = 0; i < result.total_part_count(); ++i) {
+      *stream << "  stack: |-\n";
```

Comment:
It's how we handle tracebacks from failed tests in the other test suite. It comes from how the tap13 producer in mocha wrote it. There seems to be these "almost defined" best practises for how traces should look in yamlish. Not saying its the right thing, but it is also how test.py output looks.